api-report/map.api.md

```diff
@@ -131,6 +150,16 @@ export interface ISharedMapEvents extends ISharedObjectEvents {
     (event: "clear", listener: (local: boolean, target: IEventThisPlaceHolder) => void): any;
 }
 
+// @public
+export interface ISubDirectoryCreated extends IDirectoryCreated {
+    path: string;
+}
+
+// @public
+export interface ISubDirectoryDeleted extends IDirectoryDeleted {
+    path: string;
```

Comment:
Maybe it's inline with rest of the flow, but I find it weird to have both key & path in payloads. I'd think that one field is enough. It's always relative path to the object (directory / sub-directory) where event is raised. Knowing its absolute path, one can build absolute path for deleted or created directory in whole directory.

Reply:
Replaced it with relative path.